engine/src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -149,7 +150,7 @@ object WorkflowActor {
                                initializationData: AllBackendInitializationData,
                                lastStateReached: StateCheckpoint,
                                effectiveStartableState: StartableState,
-                               workflowFinalOutputs: Set[WomValue] = Set.empty,
+                               workflowFinalOutputs: Option[CallOutputs] = None,
```

Comment:
This change was made so that the `PerformWorkflowCallback` event handling has access to the final  `CallOutputs`, which it needs to create its POST request. Previously it had access to their values but not their labels/names.